### PR TITLE
CIV-11301 dashboard notification trial arrangements

### PR DESCRIPTION
**Pull request by laarmada**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-11301

### Change description ###

Trial Arrangements Required (both parties)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseEvent/systemUpdate-R2-CUI-nonprod.json [hunk 1]
```diff
@@ -136,5 +136,11 @@
     "CaseEventID": "CREATE_DASHBOARD_NOTIFICATION_FOR_CLAIM_SETTLED_FOR_DEFENDANT1",
     "UserRole": "caseworker-civil-systemupdate",
     "CRUD": "CRU"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CREATE_DASHBOARD_NOTIFICATION_CP_TRIAL_ARRANGEMENTS_BOTH_PARTIES",
+    "UserRole": "caseworker-civil-systemupdate",
+    "CRUD": "CRU"
```
Comment (m-meulendijks-v1) [inline ccd-definition/AuthorisationCaseEvent/systemUpdate-R2-CUI-nonprod.json:144]: Either this authorisation needs to be 2, or the events below need to be one - I think something was missed out.

File: bin/utils/process-definition.sh [hunk 2]
```diff
@@ -14,11 +14,11 @@ if [[ ! -e ${definition_output_file} ]]; then
...
 fi
 
-docker run --rm --name json2xlsx \
+docker run --rm --name json2xlsx --user=$UIDcd  \
   -v ${definition_input_dir}:/tmp/ccd-definition \
   -v ${definition_output_file}:/tmp/ccd-definition.xlsx \
-  -e CCD_DEF_CASE_SERVICE_BASE_URL=${CCD_DEF_CASE_SERVICE_BASE_URL:-http://docker.for.mac.localhost:4000} \
-  -e CCD_DEF_GEN_APP_SERVICE_BASE_URL=${CCD_DEF_GEN_APP_SERVICE_BASE_URL:-http://docker.for.mac.localhost:4550} \
+  -e CCD_DEF_CASE_SERVICE_BASE_URL=${CCD_DEF_CASE_SERVICE_BASE_URL:-http://192.168.1.161:4000} \
+  -e CCD_DEF_GEN_APP_SERVICE_BASE_URL=${CCD_DEF_GEN_APP_SERVICE_BASE_URL:-http://192.168.1.161:4550} \
```
Comment (m-meulendijks-v1) [inline bin/utils/process-definition.sh:21]: I don't think we need these changes.

File: ccd-definition/AuthorisationCaseEvent/systemUpdate-R2-CUI-nonprod.json (hunk 1 above)
Comment (laarmada) [reply ccd-definition/AuthorisationCaseEvent/systemUpdate-R2-CUI-nonprod.json:144]: I have push the wrong file :) thanks max

**Review by m-meulendijks-v1 — APPROVED** [review]

File: ccd-definition/AuthorisationCaseEvent/systemUpdate-R2-CUI-nonprod.json [hunk 3]
```diff
@@ -136,5 +136,17 @@
     "CaseEventID": "CREATE_DASHBOARD_NOTIFICATION_FOR_CLAIM_SETTLED_FOR_DEFENDANT1",
     "UserRole": "caseworker-civil-systemupdate",
     "CRUD": "CRU"
+  },
+  {
+    "CaseTypeID": "CIVIL",
```
[the remark this answers is not in the document]
Comment (laarmada) [reply ccd-definition/AuthorisationCaseEvent/systemUpdate-R2-CUI-nonprod.json:141]: I have change it, forget to push the latest change.

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**